.github/workflows/s3-workflow-rollout-to-scaleway.yml

```diff
@@ -0,0 +1,69 @@
+---
+name: Rollout S3 buckets (Scaleway)
+on:  # yamllint disable-line rule:truthy
+  workflow_call:
+    inputs:
+      # Scaleway information
+      scaleway-region:
+        description: Scaleway region to deploy to
+        required: true
+        type: string
+    secrets:
+      SCALEWAY_ACCESS_KEY:
+        description: Scaleway API key ID
+        required: true
+      SCALEWAY_SECRET_KEY:
+        description: Scaleway API key secret
+        required: true
+
+jobs:
+  generate-matrix:
+    name: Generate job matrices
+    runs-on: ubuntu-latest
+    outputs:
+      buckets: ${{ steps.filtered.outputs.changes }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          sparse-checkout: |
+            s3
+            .github
+      - name: Generate Matrix
+        id: filtered
+        uses: dorny/paths-filter@v3
+        with:
+          filters: .github/s3-rollout-filters.yaml
+
+  rollout-buckets:
+    runs-on: ubuntu-latest
+    needs: [generate-matrix]
+    name: S3CMD
```

Comment:
Change this to "Rollout {{deployment}} buckets` so we can easily see what's running.

Reply:
You mean plain text {{deployment}} or the variable?
Variable is impossible because it gets generated in that job

Reply:
The variable. It should't be impossible? Yes it gets generated in the previous job, but we can use it in THIS job (which gets repeated for each deployment).